action.yml

```diff
@@ -42,5 +42,5 @@ runs:
       shell: bash
 
     - run: ${{ inputs.install-cmd }}
-      if: "${{ inputs.install-cmd }} != '' && ${{ steps.cache-venv.outputs.cache-hit }} != 'true'"
+      if: ${{ inputs.install-cmd != '' && steps.cache-venv.outputs.cache-hit != 'true' }}
```

Comment:
oh neat thing about `if` -- you don't need `${{` since it's already an evaluation context

Reply:
noice.